warehouse/admin/views/helpscout.py

```diff
@@ -45,16 +46,22 @@ def helpscout(request):
     email = (
         request.db.query(Email)
         .where(
-            Email.email.ilike(request.json_body.get("customer", {}).get("email", ""))
+            Email.email.ilike(
+                re.sub(
+                    r"\+[^)]*@",
+                    "@",
+                    request.json_body.get("customer", {}).get("email", ""),
+                )
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏I know this is only used in admin views, on already-sanitized email address strings, but is there any other weird injection risk here?

Reply:
hm, I would hope that sqlalchemy would guard us against SQL injection...

I suppose if someone sent an email from an address that would case re.sub to freak out, and helpscout didn't barf, and we loaded the page... it would make it here.